yorkie-intelligence/src/.env.deployment

```diff
@@ -0,0 +1,3 @@
+MODEL_TYPE="ollama"
+MODEL_NAME="smollm2:135m"
+API_KEY="<your-api-key>"
```

Comment:
> Need [POSIX newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) here.

Same here. I also suggest leaving `API_KEY` to empty `""`, as CodeRabbits' suggested.